website/src/components/Submission/FormOrUploadWrapper.tsx

```diff
@@ -85,7 +85,8 @@ export const FormOrUploadWrapper: FC<FormOrUploadWrapperProps> = ({
                         if (!submissionId) {
                             return { type: 'error', errorMessage: 'Please specify an ID.' };
                         }
-                        const metadataFile = editableMetadata.getMetadataTsv();
+                        const fastaId = editableSequences.getFastaIds(submissionId);
+                        const metadataFile = editableMetadata.getMetadataTsv(undefined, undefined, fastaId);
```

Comment:
this is not properly updated when segments are deleted:
`Metadata file contains 1 Fasta ids that are not present in the sequence file: PQ523728.1.L/PQ523725.1.M/PQ523721.1.S_M;`